docroot/themes/humsci/humsci_basic/src/js/shared/linked-cards/linked-cards.js

```diff
@@ -22,10 +22,26 @@
     }
 
     // Add a click event listener to each card
-    function handleClick() {
+    function handleClick(event) {
+      // Ensure clicks on "Add to Calendar" button or title do not trigger the card click
+      if (event.target.classList.contains('addtocal') || event.target.classList.contains('addtocal-title')) {
+        return;
+      }
       mainLink.click();
     }
 
+    // Stop the click propagation if the click is inside the addtocal menu links
+    function handleMenuLinkClick(event) {
+      // Prevent the event from bubbling up to the card
+      event.stopPropagation();
```

Comment:
No need to define a function here, it's just one line and we're only using it once. Let's add this as an anonymous function in the `addEventListener()` below.